src/lib.rs

```diff
@@ -185,6 +185,8 @@ mod group_map;
 mod groupbylazy;
 mod intersperse;
 #[cfg(feature = "use_std")]
+mod k_smollest;
```

Comment:
Hmm, I wonder if `itertools` will grow an `alloc` feature, as that's all a lot of these need (AFAIK).

Reply:
I agree that an `alloc` feature would be nice, as it looks like (almost?) all of those only need that (rather than all of `std`)

Reply:
Done :)